src/farva/pkg/gateway/reverseproxy_nginx.go

```diff
@@ -51,6 +51,18 @@ http {
 {{- end }}
     }
 {{ end }}
+    server {
+        listen {{ .NGINXMetricsPort }};
```

Comment:
All of the other servers should have server names, though. I'll send you an example config directly.